zamba/models/model.py

```diff
@@ -47,6 +36,25 @@ def __del__(self):
         if self.delete_tempdir:
             rmtree(self.tempdir)
 
+    def load(self):
+        if self.model_path is not None:
+            if self.model_library == 'keras':
+                import keras
```

Comment:
I think that we want `from tensorflow import keras` so that we use the version that comes with tensorflow and not force that keras is installed separately